src/utils/mtev_lfu.c

```diff
@@ -305,8 +319,12 @@ mtev_lfu_get(mtev_lfu_t *c, const char *key, size_t key_len)
     return NULL;
   }
 
-  unsigned long hash = CK_HS_HASH(&c->hash, lfu_entry_hash, key);
-  void *entry = ck_hs_get(&c->hash, hash, key);
+  struct lfu_key *tempkey = alloca(sizeof(struct lfu_key) + key_len);
```

Comment:
If key_len is not bounded, may want to put a size limit on it, lest you alloca() something infinitely huge.... like, for example:

if (key_len < 65535) /* use alloca */
else /* use malloc/free */